scrapers/WowPorn.yml

```diff
@@ -0,0 +1,22 @@
+name: "WowPorn"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - wowporn.xxx
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //meta[@property="og:title"][1]/@content
+      Date:
+        selector: //div[@class="fp-player"]//meta[@itemprop="uploadDate"]/@content
+        parseDate: 2006-01-02
+      Details: //div[@class="video-description"]/div/p/text()
```

Comment:
needs something more https://www.wowporn.xxx/feel-better-soon-with-alegra-and-cassie-fire/
```
Details:
  selector: //div[@class="video-description"]/div/p//text()
  concat: " "
```
seems to work

Reply:
Updated!